odml/tools/odmlparser.py

```diff
@@ -71,7 +71,7 @@ def write_file(self, odml_document, filename):
         report = validation.report()
         if report:
             msg += "The saved Document contains formal issues."
-            msg += " Run 'odml.Validation' to resolve them.\n%s" % report
+            msg += " Run 'odml.validation.Validation(doc)' to resolve them.\n%s" % report
```

Comment:
I'm a little confused by this.  Does running the validation resolve the errors or just collect them?  Also, we just ran through the validation and we're asking the user to rerun it?

Reply:
Validations never resolve any issues, they collect all issues and give access to the offending objects.

The validation here is run in the background and a user has no access to it. After the validation is done in the background, a user is then informed via a short report if there were minor issues with the document and that details to resolve them can be obtained by running the basic validation. The idea is basically to inform users about issues but not to enforce solving them immediately.

An option would be to print all the validation messages to the user, but that might get quite spammy and it could be that most users actually don't care about minor issues and would be annoyed by more unwanted output.

Again if there are any suggestions how to handle this differently, let me know!

Reply:
With respect to the phrasing of the message, rephrase to `Run 'odml.validation.Validation(doc)' to access/retrieve them.`